packages/composites/rocket-pool/src/config/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Requester, util } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/ea-bootstrap'
+import { ethers } from 'ethers'
+import rethAbi from '../abis/rethAbi.json'
+
+export const NAME = 'ROCKET_POOL'
+
+export const DEFAULT_ENDPOINT = 'reth'
+export const DEFAULT_ETH_USD_PROXY_ADDRESS = '0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419'
+export const RETH_TOKEN_CONTRACT = '0xae78736Cd615f374D3085123A210448E74Fc6393'
```

Comment:
Should we allow overriding this with an environment variable or input parameter? Similar to `ETH_USD_PROXY_ADDRESS`

Reply:
Another, suggestion if the testnet addresses are known is to hold a lookup for `{ [network: string]: contractAddress: string }`
(and still allow overrides)

Reply:
Note to self: this would also require extending the source to be able to come from a different chain.

Reply:
I like the override idea, I will add that in. For the lookup map, I don't know any of the other addresses so I'd rather not look into that now. Could be useful in the future though so I'll leave a comment.